draft-ietf-quic-recovery.md

```diff
@@ -1189,10 +1191,13 @@ SetLossDetectionTimer():
     loss_detection_timer.update(loss_time)
     return
 
-  // Don't arm timer if there are no ack-eliciting packets
-  // in flight and the handshake is complete.
-  if (endpoint is client with 1-RTT keys &&
-      no ack-eliciting packets in flight):
+  // Cancel the timer if there are no ack-eliciting packets
+  // in flight and the endpoint is a server. Do not cancel the
+  // timer until the client has received a Handshake or 1-RTT ACK.
+  if (no ack-eliciting packets in flight &&
+      (endpoint is server ||
+       has received Handshake ACK ||
+       has received 1-RTT ACK)):
```

Comment:
Suggestion: if !(ack-eliciting packet in flight || (endpoint is client && has not received handshake ACK && has not received 1-RTT ACK))

Reply:
As a code reviewer, I'd request a method like PeerAwaitingAddressValidation() and say 

if (no ack-eliciting packets in flight &&
     !PeerAwaitingAddressValidation()):